eng/Signing.props

```diff
@@ -5,5 +5,31 @@
     <AllowEmptySignList>true</AllowEmptySignList>
     <AllowEmptySignPostBuildList>true</AllowEmptySignPostBuildList>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <!--
+      Replace the default items to sign with the specific set we want. This allows the build to call
+      Arcade's Sign.proj multiple times for different sets of files as the build progresses.
+    -->
+    <ItemsToSign Remove="@(ItemsToSign)" />
+
+    <!-- Exclude symbol packages from getting a NuGet signature. These are never pushed to NuGet.org or
+         other feeds (in fact, that have identical identity to their non-symbol variant) -->
+    <DownloadedSymbolPackages Include="$(ArtifactsDir)**\*.symbols.nupkg" />
+    <DownloadedSymbolPackagesWithoutPaths Include="@(DownloadedSymbolPackages->'%(Filename)%(Extension)')" />
+    <FileSignInfo Include="@(DownloadedSymbolPackagesWithoutPaths->Distinct())" CertificateName="None" />
+    
+    <!-- Update existing defaults from arcade that were using Microsoft400 to use the .NET-specific cert -->
+    <FileExtensionSignInfo Update="@(FileExtensionSignInfo->WithMetadataValue('CertificateName','Microsoft400'))" CertificateName="MicrosoftDotNet500" />
+    <StrongNameSignInfo Update="@(StrongNameSignInfo->WithMetadataValue('CertificateName','Microsoft400'))" CertificateName="MicrosoftDotNet500" />
+    <FileSignInfo Update="@(FileSignInfo->WithMetadataValue('CertificateName','Microsoft400'))" CertificateName="MicrosoftDotNet500" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsPackable)' == 'true'">
+    <ItemsToSignWithPaths Include="$(ArtifactsDir)**\*.nupkg" />
+    <ItemsToSignWithoutPaths Include="@(ItemsToSignWithPaths->'%(Filename)%(Extension)')" />
+    <ItemsToSignPostBuild Include="@(ItemsToSignWithoutPaths->Distinct())" Condition="'$(PostBuildSign)' == 'true'" />
+    <ItemsToSign Include="@(ItemsToSignWithPaths->Distinct())" Condition="'$(PostBuildSign)' != 'true'" />
+  </ItemGroup>
```

Comment:
I don't think this is required but how would that work? I don't think the `IsPackable` property is set in Sign.proj (lives in Arcade) which imports this file.

Reply:
Since we are doing incremental servicing in this repo, we will enable the IsPackable property in all the csprojs we want to publish. At the moment, all of them have it set to true.

Are you saying the csprojs do not interact with Signing.props and the IsPackable property won't reach here?

Reply:
> Are you saying the csprojs do not interact with Signing.props and the IsPackable property won't reach here?

Correct. Signing.props is imported by Sign.proj, not by the individual projects in this repo.

Reply:
In runtime, the condition used for this Itemgroup is PrepareArtifacts==true. I couldn't find that property in arcade. I only found it being set in runtime, in installer. If we had to set that property to true, we would have to specify it in the ymls.

But stepping back a bit, as you said, maybe all these changes are not necessary, except the one that changes the Certificate using the method you mentioned above.